package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "aristurtle.monogame.github.io",
+    "name": "monogame.github.io",
```

Comment:
Not sure why the package lock keeps changing like this 😅.  I think it picks up from the directory I clone it to to keep it separate from the main repo clone.